### PR TITLE
Update plugin-install-manager, move jar to /opt

### PR DESCRIPTION
**Pull request by timja**
Fixes https://github.com/jenkinsci/azure-ad-plugin/issues/121

See https://github.com/jenkinsci/azure-ad-plugin/issues/121#issuecomment-867919247

### CODE REVIEW

**Comment by timja** [comment]
```
[2021-06-25T08:56:18.669Z] # -- command failed --

[2021-06-25T08:56:18.669Z] # status : 1

[2021-06-25T08:56:18.669Z] # output (2 lines):

[2021-06-25T08:56:18.669Z] #   touch: cannot touch '/var/jenkins_home/copy_reference_file.log': Permission denied

[2021-06-25T08:56:18.669Z] #   Can not write to /var/jenkins_home/copy_reference_file.log. Wrong volume permissions?
```

**Review by MarkEWaite — COMMENTED** [review]
Don't we need an entry in a changelog that alerts users that the location has changed?

File: 8/centos/centos7/hotspot/Dockerfile [hunk 1]
```diff
@@ -51,10 +51,10 @@ RUN curl -fsSL https://github.com/krallin/tini/releases/download/${TINI_VERSION}
 
 # jenkins version being bundled in this docker image
 ARG JENKINS_VERSION
-ENV JENKINS_VERSION ${JENKINS_VERSION:-2.235.4}
+ENV JENKINS_VERSION ${JENKINS_VERSION:-2.299}
```
Comment (MarkEWaite) [inline 8/centos/centos7/hotspot/Dockerfile:54]: As far as I know, this update is not harmful, but it also does not alter anything in the build process.  The Jenkins Docker images are always built by providing environment values for `JENKINS_VERSION` and `JENKINS_SHA`

**Review by MarkEWaite — APPROVED** [review]
Approved on the assumption that tests need to pass before merge and we probably should write somewhere that the location of the jar file has changed.

Fascinating bug report and surprising condition that jar files in /usr/lib were added to the class path.

File: 8/centos/centos7/hotspot/Dockerfile (hunk 1 above)
Comment (timja) [reply 8/centos/centos7/hotspot/Dockerfile:54]: Oh this was accidental

File: 8/centos/centos7/hotspot/Dockerfile [hunk 2]
```diff
@@ -51,10 +51,10 @@ RUN curl -fsSL https://github.com/krallin/tini/releases/download/${TINI_VERSION}
...
 
 # jenkins.war checksum, download will be validated using it
-ARG JENKINS_SHA=e5688a8f07cc3d79ba3afa3cab367d083dd90daab77cebd461ba8e83a1e3c177
+ARG JENKINS_SHA=9663a14158588eb470a024ad941f6780945c6383b5a82dda4ef2d15dac4f7214
```
Comment (timja) [inline 8/centos/centos7/hotspot/Dockerfile:57]:
```suggestion
ARG JENKINS_SHA=e5688a8f07cc3d79ba3afa3cab367d083dd90daab77cebd461ba8e83a1e3c177
```

File: 8/centos/centos7/hotspot/Dockerfile (hunk 1 above)
Comment (timja) [reply 8/centos/centos7/hotspot/Dockerfile:54]:
```suggestion
ENV JENKINS_VERSION ${JENKINS_VERSION:-2.235.4}
```

File: 8/centos/centos8/hotspot/Dockerfile [hunk 3]
```diff
@@ -51,10 +51,10 @@ RUN curl -fsSL https://github.com/krallin/tini/releases/download/${TINI_VERSION}
 
 # jenkins version being bundled in this docker image
 ARG JENKINS_VERSION
-ENV JENKINS_VERSION ${JENKINS_VERSION:-2.235.4}
+ENV JENKINS_VERSION ${JENKINS_VERSION:-2.299}
 
 # jenkins.war checksum, download will be validated using it
-ARG JENKINS_SHA=e5688a8f07cc3d79ba3afa3cab367d083dd90daab77cebd461ba8e83a1e3c177
+ARG JENKINS_SHA=9663a14158588eb470a024ad941f6780945c6383b5a82dda4ef2d15dac4f7214
```
Comment (timja) [inline 8/centos/centos8/hotspot/Dockerfile:57]:
```suggestion
ARG JENKINS_SHA=e5688a8f07cc3d79ba3afa3cab367d083dd90daab77cebd461ba8e83a1e3c177
```

File: 8/centos/centos8/hotspot/Dockerfile [hunk 4]
```diff
@@ -51,10 +51,10 @@ RUN curl -fsSL https://github.com/krallin/tini/releases/download/${TINI_VERSION}
 
 # jenkins version being bundled in this docker image
 ARG JENKINS_VERSION
-ENV JENKINS_VERSION ${JENKINS_VERSION:-2.235.4}
+ENV JENKINS_VERSION ${JENKINS_VERSION:-2.299}
```
Comment (timja) [inline 8/centos/centos8/hotspot/Dockerfile:54]:
```suggestion
ENV JENKINS_VERSION ${JENKINS_VERSION:-2.235.4}
```

**Pull request merged**